### PR TITLE
Add error checks in Actions

### PR DESCRIPTION
**Pull request by kenjis**
- add User null check
- add check to see if the email sent

### CODE REVIEW

File: src/Authentication/Actions/Email2FA.php [hunk 1]
```diff
@@ -70,13 +75,17 @@ public function handle(IncomingRequest $request)
 
         // Send the user an email with the code
         helper('email');
-        $email = emailer();
-        $email->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '')
+        $emailer = emailer();
+        $ret     = $emailer->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '')
```
Comment (MGatner) [inline src/Authentication/Actions/Email2FA.php:79]: What is "ret"? If this is short for "return" can you just use the whole word?

File: src/Authentication/Actions/Email2FA.php [hunk 2]
```diff
@@ -70,13 +75,17 @@ public function handle(IncomingRequest $request)
 
         // Send the user an email with the code
         helper('email');
-        $email = emailer();
-        $email->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '')
+        $emailer = emailer();
```
Comment (MGatner) [inline src/Authentication/Actions/Email2FA.php:78]: Is the intermediate `$emailer` variable necessary?

**Review by MGatner — CHANGES_REQUESTED** [review]

File: src/Authentication/Actions/Email2FA.php (hunk 1 above)
Comment (kenjis) [reply src/Authentication/Actions/Email2FA.php]: Yes, and fixed!

File: src/Authentication/Actions/Email2FA.php (hunk 2 above)
Comment (kenjis) [reply src/Authentication/Actions/Email2FA.php]: Removed.

File: src/Authentication/Actions/EmailActivator.php [hunk 3]
```diff
@@ -38,13 +43,17 @@ public function show(): string
 
         // Send the email
         helper('email');
-        $email = emailer();
-        $email->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '')
+        $emailer = emailer();
+        $ret     = $emailer->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '')
```
Comment (MGatner) [inline src/Authentication/Actions/EmailActivator.php:47]:
```suggestion
        $return = emailer()->setFrom(setting('Email.fromEmail'), setting('Email.fromName') ?? '')
```

File: src/Authentication/Actions/EmailActivator.php [hunk 4]
```diff
@@ -38,13 +43,17 @@ public function show(): string
...
             ->setTo($user->getAuthEmail())
             ->setSubject(lang('Auth.emailActivateSubject'))
             ->setMessage(view(setting('Auth.views')['action_email_activate_email'], ['code' => $code]))
             ->send();
 
+        if ($ret === false) {
```
Comment (MGatner) [inline src/Authentication/Actions/EmailActivator.php:53]:
```suggestion
        if ($return === false) {
```

**Review by MGatner — CHANGES_REQUESTED** [review]
Some song second verse.

**Review by MGatner — APPROVED** [review]

**Pull request merged**